src/main/java/io/cosmosoftware/kite/instrumentation/Profile.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) CoSMo Software Consulting Pte. Ltd. - All Rights Reserved
+ */
+
+package io.cosmosoftware.kite.instrumentation;
+
+import io.cosmosoftware.kite.exception.KiteTestException;
+import io.cosmosoftware.kite.report.Status;
+
+import javax.json.JsonObject;
+
+public class Profile {
+
+  private final String nit;
+  private final String command;
+  private final int delay;
+  private final int packetloss;
+  private final int corrupt;
+  private final int duplicate;
+  private final int bandwidth;
+
+  public Profile(JsonObject jsonObject) throws Exception {
+    this.nit = "enp0s8";
+    this.delay =  jsonObject.getInt("delay", 0);
+    this.packetloss = jsonObject.getInt("packetloss", 0);
+    this.corrupt = jsonObject.getInt("corrupt", 0);
+    this.duplicate = jsonObject.getInt("duplicate", 0);
+    this.bandwidth = jsonObject.getInt("bandwidth", 0);
+    this.command = jsonObject.containsKey("command") ? jsonObject.getString("command") : this.setCommand();
+  }
+
+  private String setCommand() throws Exception {
+    String command;
+    String egress_command = "";
+    String ingress_command = "";
+    command = "sudo ip link add ifb0 type ifb || true && sudo ip link set up dev ifb0 || true && sudo tc qdisc add dev " + this.nit + " ingress || true && sudo tc filter add dev " + this.nit + " parent ffff: protocol ip u32 match u32 0 0 action mirred egress redirect dev ifb0 || true && ";
```

Comment:
make this a bit more comprehensive for people who might work on your code after. Something like:
```
final String doThis = "this command do this";
final String doThat = "this command do that";
...

String command = doThis + " || " + doThat + " || " + ...
```